Script Examples/Python/testing_example.py

```diff
@@ -1,19 +1,17 @@
 """
-Example of unit testing.
+Examples of unit testing.
 """
 import unittest
 import time
-import core.items
```

Comment:
This import will be needed. In your next commit, please add the signoff.

Reply:
Thank you for your hint, the imports are modified. I will sign as soon as I can.